charts/cloudserver-front/values.yaml

```diff
@@ -29,6 +29,12 @@ image:
   tag: 0.2.1
   pullPolicy: IfNotPresent
 
+proxy:
+  # If you want to use an http proxy, add the respective endpoints after
+  # 'http:' and/or 'https:'
```

Comment:
Maybe add a note explaining that, if a HTTP proxy is set but no HTTPS proxy, the HTTP one will be used for HTTPS traffic anyway.